cfpb/cfgov-refresh
# Adds initial accessibility tracking custom gtm tag

This adds code for tracking the skip link button that shows when navigating the site using the tab key. This will have a corresponding tag in GTM.

## Additions

- Add accessibility interactions GTM tracking code listener.

## Changes

- Add dashes in jsdoc comment.

## Testing

1. Add `require( '../../apps/analytics-gtm/js/accessibility-interactions-listeners' );` to `common.js`
2. Run `npm run gulp clean && npm run gulp build`
3. Visit http://localhost:8000/?debug-gtm=true
4. Open the dev console
5. Hit the tab key and see that a tracking event logs to the console.
